samples/nrf9160/ciphersuites/src/main.c

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <string.h>
+
+#include <zephyr.h>
+#include <net/socket.h>
+#include <net/tls_credentials.h>
+#include <modem/lte_lc.h>
+#include <modem/modem_key_mgmt.h>
+
+#define HTTPS_PORT 443
+
+#define HTTPS_HOSTNAME "example.com"
+
+#define TLS_SEC_TAG 42
+
+/* Certificate for `example.com` */
+static const char cert[] = {
+	#include "../cert/DigiCertGlobalRootCA.pem"
+};
+
+BUILD_ASSERT(sizeof(cert) < KB(4), "Certificate too large");
+
+struct lookup_ciphersuite {
+	char *name;
+	uint16_t value;
+	bool supported;
+};
+
+/* Taken from the IANA register */
+static struct lookup_ciphersuite ciphersuites[] = {
+		{"TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384", 0xC024},
+		{"TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA", 0xC00A},
+		{"TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256", 0xC023},
+		{"TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA", 0xC009},
+		{"TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA", 0xC014},
+		{"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256", 0xC027},
+		{"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA", 0xC013},
+		{"TLS_PSK_WITH_AES_256_CBC_SHA", 0x008D},
+		{"TLS_PSK_WITH_AES_128_CBC_SHA256", 0x00AE},
+		{"TLS_PSK_WITH_AES_128_CBC_SHA", 0x008C},
+		{"TLS_PSK_WITH_AES_128_CCM_8", 0xC0A8},
+		{"TLS_EMPTY_RENEGOTIATIONINFO_SCSV", 0x00FF},
+#if defined(CONFIG_EXTENDED_CIPHERSUITE_LIST)
+		{"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", 0xC02B},
+		{"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384", 0xC030},
+		{"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256", 0xC02F},
+#endif
+};
+
+/* Provision certificate to modem */
+int cert_provision(void)
+{
+	int err;
+	bool exists;
+	int mismatch;
+
+	/* It may be sufficient for the application to check whether the correct
+	 * certificate is provisioned with a given tag directly using modem_key_mgmt_cmp().
+	 * Here, for the sake of the completeness, we check that a certificate exists
+	 * before comparing it with what we expect it to be.
+	 */
+	err = modem_key_mgmt_exists(TLS_SEC_TAG, MODEM_KEY_MGMT_CRED_TYPE_CA_CHAIN, &exists);
+	if (err) {
+		printk("Failed to check for certificates, err %d\n", err);
+		return err;
+	}
+
+	if (exists) {
+		mismatch = modem_key_mgmt_cmp(TLS_SEC_TAG,
+						  MODEM_KEY_MGMT_CRED_TYPE_CA_CHAIN,
+						  cert, strlen(cert));
```

Comment:
Please fix alignment

Reply:
Looks fine on my end

Reply:
Certainly lines 75-76 are off:
![image](https://user-images.githubusercontent.com/22167799/166691013-cedea709-13fb-4885-994d-7712890296de.png)

Reply:
@MirkoCovizzi  Please check in browser.

Reply:
@eivindj-nordic I'm using the auto-formatting of clang taken from the repo `.clang-format` config, Looks correct on my machine.